crates/burn-import/src/burn/node/sum.rs

```diff
@@ -0,0 +1,108 @@
+use super::{Node, NodeCodegen};
+use crate::burn::{Scope, TensorType, Type};
+
+use burn::record::PrecisionSettings;
+use proc_macro2::TokenStream;
+use quote::quote;
+
+#[derive(Debug, Clone, new)]
+pub struct SumNode {
+    pub inputs: Vec<TensorType>,
+    pub output: TensorType,
+}
+
+impl<PS: PrecisionSettings> NodeCodegen<PS> for SumNode {
+    fn output_types(&self) -> Vec<Type> {
+        vec![Type::Tensor(self.output.clone())]
+    }
+
+    fn input_types(&self) -> Vec<Type> {
+        self.inputs
+            .iter()
+            .map(|t| Type::Tensor(t.clone()))
+            .collect()
+    }
+
+    fn forward(&self, scope: &mut Scope, node_position: usize) -> TokenStream {
+        let inputs = self
+            .inputs
+            .iter()
+            .map(|t| scope.tensor_use_owned(t, node_position));
+
+        let output = &self.output.name;
+
+        quote! {
+            let #output = #(#inputs)+*;
+        }
+    }
+
+    fn into_node(self) -> Node<PS> {
+        Node::Sum(self)
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use burn::record::FullPrecisionSettings;
+
+    use super::*;
+    use crate::burn::{
+        graph::BurnGraph,
+        node::{sum::SumNode, test::assert_tokens},
+        TensorType,
+    };
+
+    #[test]
+    fn test_codegen_concat() {
```

Comment:
test_codegen_concat => test_codegen_sum

Reply:
Oh, I missed that! That's what I get for the mindless copying:)